action.yml

```diff
@@ -16,6 +16,14 @@ inputs:
     description: "Path of the directory containing your site"
     required: false
     default: "."
+  deno-version:
+    description: "The Deno version to use when deno.json/deno.jsonc/deno.lock exists"
+    required: false
+    default: "1.x"
```

Comment:
Can we reuse the same approach as for other package managers where we use the `package-manager` input rather than introducing a new option just for one package manager?

```yml
    package-manager: deno@1
```

(Relatedly, we should update the `description` for `package-manager` in this case.)